feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap

```diff
@@ -1,134 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature eager loading should import appropriate modules 1`] = `""`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature eager loading should import appropriate modules 1`] = `""`;
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature general setup should add the feature using the lazy loading syntax 1`] = `""`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup should add the feature using the lazy loading syntax 1`] = `""`;
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature general setup styling should create a proper scss file 1`] = `
-"@import "../../styles-config";
-@import "@spartacus/estimated-delivery-date";
-"
-`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup should add the feature using the lazy loading syntax 2`] = `
```

Comment:
I'm not sure if there's a mistake in the text or the output. It mentions that it should be loaded using lazy loading syntax, but I don't see it.

Reply:
we are lazy loading estimated delivery date via cart we have introduced a wrapper for the same